src/js/components/landing/recentActivity/RecentActivityTable.jsx

```diff
@@ -151,10 +151,16 @@ export default class RecentActivityTable extends React.Component {
 				reportingDateString,
 				userName,
 				item.last_modified,
-				<Status.SubmissionStatus status={item.rowStatus} certified={item.publish_status !== 'unpublished'} />,
-				<DeleteLink submissionId={item.submission_id} index={index} warning={this.deleteWarning.bind(this)} confirm={deleteConfirm} reload={this.reload.bind(this)} item={item} account={this.state.account}/>
+				<Status.SubmissionStatus status={item.rowStatus} certified={item.publish_status !== 'unpublished'} />
 			];
 
+			if(item.publish_status === "unpublished") {
```

Comment:
Yes, only unpublished submissions can be deleted. If they've ever been certified they can no longer be deleted.